azurerm/azurerm_container_registry/simple/main.tf

```diff
@@ -0,0 +1,40 @@
+# Summary: A simple Azure Container Registry
+
+# Documentation: https://www.terraform.io/docs/language/settings/index.html
+terraform {
+  required_version = ">= 0.14.0"
+  required_providers {
+    azurerm = {
+      source  = "hashicorp/azurerm"
+      version = "~> 2.0"
+    }
+  }
+}
+
+# Documentation: https://www.terraform.io/docs/language/values/variables.html
+variable "azure_subscription_id" {
+  type = string
+}
+
+# Documentation: https://www.terraform.io/docs/language/providers/requirements.html
+# Documentation: https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs
+provider "azurerm" {
+  features {}
+
+  subscription_id = var.azure_subscription_id
+}
+
+# Resource Group
+# Documentation: https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/resource_group
+resource "azurerm_resource_group" "changeme_simple_container_registry_resource_group" {
+  name     = "changeme-simple-kubernetes-cluster-resource-group"
+  location = "West Europe"
+}
+
+# Container Registry within the Resource Group
+# Documentation: https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/container_registry
+resource "azurerm_container_registry" "changeme_simple_container_registry" {
+  name                = "changemesimplecontainerregistry"
+  resource_group_name = azurerm_resource_group.changeme_simple_container_registry_resource_group.name
+  location            = azurerm_resource_group.changeme_simple_container_registry_resource_group.location
+}
```

Comment:
This resource caused a CI error when Applying to Azure.
See: https://github.com/ContainerSolutions/terraform-examples/runs/3083782279?check_suite_focus=true
```
azurerm_container_registry.changeme_simple_container_registry: Creating...
╷
│ Error: `storage_account_id` must be specified for a Classic (unmanaged) Sku.
│ 
│   with azurerm_container_registry.changeme_simple_container_registry,
│   on main.tf line 36, in resource "azurerm_container_registry" "changeme_simple_container_registry":
│   36: resource "azurerm_container_registry" "changeme_simple_container_registry" ***
│ 
```

There's something about it in the docs: https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/container_registry
Should we add:
```
sku                 = "Standard"
```